lit_llama/lora.py

```diff
@@ -35,7 +37,7 @@ def __init__(
         self.merge_weights = merge_weights
 
 
-class MergedLinear(nn.Linear, LoRALayer):
+class MergedLinear(bnb.nn.Linear8bitLt, LoRALayer):
```

Comment:
See: https://github.com/huggingface/peft/blob/df71b84341ae1ab3bc9b0d5f906d7a524850b63b/src/peft/tuners/lora.py#L539 they removed this from the original implementation

not sure whether we need to do the same.